### ISSUE TITLE
Error in vector(type, length) :    vector: cannot make a vector of mode 'NULL'

### ISSUE THREAD

**Issue by LABurns** [Ram-N/weatherData#32]
Code:
KORPORTL52 <- getWeatherForDate("KORPORTL52", start_date="2016-06-01",
                       end_date = "2016-06-28",
                       station_type = "ID",
                       opt_detailed = TRUE,
                       opt_all_columns = TRUE)

Returns error: Error in vector(type, length) :    vector: cannot make a vector of mode 'NULL'

There is indeed data in this date range, just won't be written to df.

**Comment by Ram-N** [comment]
Thanks @LABurns. I will look into it right away and fix it soon

**Comment by Ram-N** [comment]
Hi: 

WHen I run your code, it works for me with no errors. I believe that this is because in the github version of the code, I have fixed this issue. However, I'd like you to test and confirm that this is indeed true.

Could you try downloading from Github:
(You also need devtools from CRAN. Here are the steps:)

```
install.packages("devtools")
library("devtools")
install_github("Ram-N/weatherData")

```

Please let me know how it goes.

**Comment by Ram-N** [comment]
This was fixed in v0.5.0 and is now on CRAN.
